index.d.ts

```diff
@@ -20,6 +20,10 @@ declare module 'jwks-rsa' {
       rsaPublicKey?: string;
     }
 
+    interface Headers {
+      [key: string]: any;
```

Comment:
shouldn't this be `string` instead of `any`? What scenario am I not seeing here?

Reply:
I copied this interface from the `request` module (see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c8079029312d0afeb62ce04c18a1a471a92bd7ca/types/request/index.d.ts#L342-L344 )
 Actually idk why it says `any` there. Maybe because they want to support numbers and `undefined` too and cast them internally somehow? Anyway, I changed it to `string` now. Makes sense in this context.